project/OSGi.scala

```diff
@@ -17,15 +17,15 @@ object OSGi {
   // a pain. Create bundles but publish them to the normal .../jars directory.
   def osgiSettings =
     defaultOsgiSettings ++ Seq(
-      Compile / packageBin := {
+      /* Compile / packageBin := {
```

Comment:
?

Reply:
I had problems publishing locally because the reproducible builds plugin corrupted my jars (at least I think that was it) but I didn't intend to commit this though.